components/cmake/build_scream.cmake

```diff
@@ -9,10 +9,18 @@ function(build_scream)
 
     include(${CMAKE_SOURCE_DIR}/cmake/common_setup.cmake)
 
+    # Transfer CIME-set flags into Cmake-style
     set (CMAKE_C_FLAGS ${CFLAGS})
     set (CMAKE_CXX_FLAGS ${CXXFLAGS})
     set (CMAKE_Fortran_FLAGS ${FFLAGS})
 
+    # SCREAM manages its own kokkos settings. We can think about
+    # removing this once the cime cmake_macros aren't using obsolete
+    # Kokkos settings like KOKKOS_OPTIONS.
+    if (KOKKOS_OPTIONS)
+      unset(KOKKOS_OPTIONS)
```

Comment:
I'm confused. The CIME mach file for quartz sets this var, but then we unset it? Is it not used then? If so, why do we need this var?